examples/Bremsstrahlung/include/simulation_defines/param/bremsstrahlung.param

```diff
@@ -0,0 +1,137 @@
+/**
+ * Copyright 2016-2017 Heiko Burau
+ *
+ * This file is part of PIConGPU.
+ *
+ * PIConGPU is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * PIConGPU is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with PIConGPU.
+ * If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+namespace picongpu
+{
+namespace particles
+{
+namespace bremsstrahlung
+{
+
+/** enable bremsstrahlung */
+# ifndef ENABLE_BREMSSTRAHLUNG
```

Comment:
[style:](https://github.com/ComputationalRadiationPhysics/contributing/blob/663e74126ce168667a0f85400d907c0cbd4f893c/codingGuideLines/cpp.md#7-preprocessor), naming and comment:
```C++
/** disable bremsstrahlung by default */
#ifndef PARAM_BREMSSTRAHLUNG
#    define PARAM_BREMSSTRAHLUNG 0
#endif
```